eng/Version.Details.xml

```diff
@@ -16,6 +16,38 @@
       <Uri>https://github.com/dotnet/wcf</Uri>
       <Sha>7f504aabb1988e9a093c1e74d8040bd52feb2f01</Sha>
     </Dependency>
+    <Dependency Name="runtime.linux-arm64.Microsoft.NETCore.Runtime.ObjWriter" Version="1.0.0-alpha.1.21630.1">
```

Comment:
We should have things set up for Arcade to auto update this. We'll see if it works next time we make an objwriter change.

Also, this would be prettier if there was a lineup package, but Mono's LLVM packages don't have lineup packages and I didn't spend time inventing how to do it with the build infra there.